apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -114,9 +121,10 @@ export default function teacherSections(state=initialState, action) {
   if (action.type === SET_SECTIONS) {
     const sections = action.sections.map(section =>
       sectionFromServerSection(section));
+    const prevSectionIds = action.reset ? [] : state.sectionIds;
     return {
       ...state,
-      sectionIds: state.sectionIds.concat(sections.map(section => section.id)),
+      sectionIds: prevSectionIds.concat(sections.map(section => section.id)),
```

Comment:
might be good to get some test coverage of this reset behavior in teacherSectionsReduxTest.js